app/Casts/AsJob.php

```
@@ -0,0 +1,189 @@
+            $this->getCompensation($attributes),
+            $this->getStack($attributes),
+            $this->getDescription($attributes),
+        );
+    }
+
+    /**
+     * Prepare the given value for storage.
+     *
+     * @param  array<string, mixed>  $attributes
+     */
+    public function set(Model $model, string $key, mixed $value, array $attributes): mixed
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$model'. (undefined)

(UnusedFormalParameter)